opsgenie_alert_tagging.rb

```diff
@@ -0,0 +1,4 @@
+require './lib/opsgenie_alert_tagging'
```

Comment:
I would be good to make this an executable. Will also need to change its permissions

```suggestion
#!/usr/bin/env ruby
require './lib/opsgenie_alert_tagging'
```